doc/examples/features_detection/plot_haar.py

```diff
@@ -0,0 +1,101 @@
+"""
+============================
+Haar-like feature descriptor
+============================
+
+Haar-like feature descriptors are a set of simple features which was used in
+the first real-time face detector [1]_. Indeed, these features can be
```

Comment:
Thanks for the example---I think a visual aid will really help readers to understand what these features are all about.

The 2nd sentence of this paragraph doesn't say much to me; could you clarify?

Also, can you try and give an intuitive description of what Haar features are, when they are typically detected, etc.?  This is where we get to educate and draw in the reader.

Reply:
Looks good to me :)

I would also add that it can be computed in a constant time on any scale
which is a great advantage over other descriptors.

Reply:
Remove "Indeed", just looks better without it :)

And please, address my comment that haar features can be computed on any scale
in constant time and cite the original Viola Jones paper next to that sentence.